### ISSUE TITLE
plotting coverage from long reads

### ISSUE THREAD

**Issue by KamilSJaron** [ryanlayer/samplot#108]
Hello,

big fan. I love the idea of samplot, well done!

I managed to make plots from Illumina data that were correct. However, when I did one with PacBio, it seems to fail to plot the coverage (looking at the examples it seems it should work, not sure where is the problem).

The plot:

![ctg148_901791_902385_DEL](https://user-images.githubusercontent.com/8181573/85847346-1f3d1000-b79f-11ea-82a2-306ca3319944.png)

samplot -v
> samplot 1.0.12

and the plotting command was 

```
samplot plot -n Tdi_PB -b data/1_Tdi/pacbio_1Tdi/mapped_to_pb_asm00_sorted.bam -o data/sandbox/ctg148_901791_902385_DEL.png -s 901791 -e 902385 -c ctg148 -a -t DEL
```

**Comment by jbelyeu** [comment]
Hmm. Not certain what's going on here. Could you try running the test suite from the github repository? You'll need to download the tests. The following commands will do that and allow you to get the development fork of the repository which have the most up-to-date version of the tests.
```
git clone https://github.com/ryanlayer/samplot.git
cd samplot/
bash runtests.sh
```

This should produce a set of new images in the `samplot/test/func/` directory. See if there's a file in that directory named `longread_del.png` and if it has a coverage plot. If it renders correctly, then the problem must be related to the way samplot is interacting with you data specifically. That image should look like this:
![longread_del](https://user-images.githubusercontent.com/6629063/85906077-799f9600-b7ca-11ea-8f27-6598afbee486.png)

I'd be happy to look into this directly if subsetting your data file and sharing the subset is an option, but if not I can certainly work with you to try to figure out what's going on.

**Comment by KamilSJaron** [comment]
I run the test and indeed, I managed to recover exactly the same figure, hence it must be something with my data.

I extracted `ctg148` with header from my bam file (`samtools view -hb -o Tdi_PB_ctg148.bam mapped_to_pb_asm00_sorted.bam ctg148`) and zipped it (GitHub allows only some file suffixes), here it is:

[Tdi_PB_ctg148.bam.zip](https://github.com/ryanlayer/samplot/files/4842599/Tdi_PB_ctg148.bam.zip)

Cheers!

**Comment by jbelyeu** [comment]
This issue was caused by improper reformatting of the chromosome name. It's fixed in the upcoming version 1.0.15 which is in conda testing right now. Should be available in the next day or so. Thanks for the heads up!

**Comment by KamilSJaron** [comment]
Awesome @jbelyeu, thanks for fixing it so quick :-)
